src/components/views/JobForm.js

```diff
@@ -97,11 +105,65 @@ class JobForm extends React.Component {
               footer={this.renderPaneFooter()}
               id="pane-job-form"
               firstMenu={this.renderFirstMenu()}
-              paneTitle={<FormattedMessage id="ui-local-kb-admin.job.newJob" />}
+              paneTitle={<FormattedMessage id={`ui-local-kb-admin.job.new${format}Job`} />}
             >
               <TitleManager record={create}>
                 <form>
                   <div className={css.jobForm}>
+                    {format === 'KBART' &&
+                      <>
+                        <Row>
+                          <Col xs={12}>
```

Comment:
Remember, rendering a `Row` with a single `Col` of `xs={12}` does nothing, so they can both be removed (throughout this new code).

Reply:
Removed